docs/GraFx-Developers/connectors/authorization-for-connectors/index.md

```diff
@@ -193,7 +193,7 @@ Each authorization type requires a specific JSON schema. The `grant_type` for OA
 }
 ```
 
-#### OAuth JWT Bearer Token
+#### OAuth 2.0 JWT Bearer Token
 ```typescript
```

Comment:
Please remove the actual example from this section as we're going to change the approach( Or just replace example with statement - "Depends on the connector definition")